### PR TITLE
Add filetype

### PR DESCRIPTION
**Pull request by druvus**

### CODE REVIEW

File: recipes/filetype/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - examples
+    - filetype
+    - filetype.types
+
+about:
+  home: https://github.com/h2non/filetype.py
+  license: MIT
+  license_family: MIT
+  license_file: ''
```
Comment (bgruening) [inline recipes/filetype/meta.yaml]: You can add here `LICENSE`

Comment (druvus) [reply recipes/filetype/meta.yaml]: The license file is not distributed in the pypi compressed file. Is it better to use the github source instead in this case?

Comment (mwcraig) [reply recipes/filetype/meta.yaml]: @druvus -- please see https://conda-forge.org/docs/meta.html#packaging-the-license-manually for how to manually add the license to the conda package.

Comment (druvus) [reply recipes/filetype/meta.yaml]: thanks @mwcraig! License file is now manually added.

**Comment by mwcraig** [comment]
Thanks for the contribution, @druvus, merging

**Pull request merged**